README.md

```diff
@@ -3,5 +3,123 @@ Babbage
 
 Repository for ONS Website Babbage
 
-#### Dependencies
+Babbage is a bespoke service for translating JSON into HTML. The Zebedee-Reader reads .json files off the server for passing to Babbage. Babbage then translates them into the .html files that are used in the website.
 
+Babbage contains two main areas of functionality, as follows:
+
+1. It creates the HTML files for the pages on the website.
+2. It creates the HTML files for the website publications in the publishing system [Florence](https://github.com/ONSdigital/florence)
+
+### Getting started
+
+In the babbage repo do one of the following:
+
+* To run Babbage for the website functionality use this command
+
+```shell script
+./run.sh
+```
+
+* To run Babbage for the publishing functionality use this command
+
+```shell script
+./run-publishing.sh
+```
+
+### Dependencies
+
+Babbage runs independently. However, in order to run it locally in its publishing mode, with the other required services for that, there is a stack that can be used in dp-compose:
+
+[Homepage publishing](https://github.com/ONSdigital/dp-compose/tree/main/v2/stacks#homepage-publishing)
+
+### Configuration
+
+| Environment variable          | Default                | Description
+| ------------------------------| -----------------------|-------------------------------------------------------------
+| CONTENT_SERVICE_MAX_CONNECTION| 50                     | The maximum number of connections Babbage can make to the content service
+| CONTENT_SERVICE_URL           | http://localhost:8082  | The URL to the content service (zebedee)
+| ELASTIC_SEARCH_SERVER         | localhost              | The elastic search host and port (The http:// scheme prefix is added programmatically)
+| ELASTIC_SEARCH_CLUSTER        |                        | The elastic search cluster
+| ENABLE_CACHE                  | N                      | Switch to use (or not) the cache
+| ENABLE_COVID19_FEATURE        |                        | Switch to use (or not) the covid feature
+| ENABLE_METRICS                | N                      | Switch to collect (or not) metrics about cache expiry times
+| HIGHCHARTS_EXPORT_SERVER      | http://localhost:9999/ | The URL to the highcharts export server
+| IS_PUBLISHING                 | N                      | Switch to use (or not) the publishing functionality
+| MAP_RENDERER_HOST             | http://localhost:23500 | The URL to the map renderer
+| METRICS_PORT                  | 1234                   | The port for the metrics URL
+| REDIRECT_SECRET               | secret                 | The code for the redirect
+| TABLE_RENDERER_HOST           | http://localhost:23300 | The URL to the table renderer
+
+### Metrics
+
+To see the metrics note that the ENABLE_METRICS and ENABLE_CACHE values must be set to Y when babbage starts up. Then call the following command while babbage is running:
```

Comment:
"To see the metrics, note" - I think needs comma.

Reply:
done 👍